tests/unit/ConsentContinuationOut.spec.ts

```diff
@@ -153,29 +149,29 @@ describe('Consent to Continuation Out view', () => {
   })
 
   it('saves draft consent to continuation out properly', async () => {
-    // mock "save draft" endpoint (garbage response data - we aren't testing that)
-    sinon.stub(axios, 'post')
-      .withArgs('businesses/CP1234567/filings?draft=true')
-      .returns(new Promise(resolve =>
-        resolve({
-          data: {
-            filing: {
-              business: {},
-              header: { filingId: 456 },
-              consentContinuationOut: { details: 'test' },
-              annualReport: {}
-            }
-          }
-        })
-      ))
+    // mock "has pending tasks" legal service
+    jest.spyOn(LegalServices, 'hasPendingTasks').mockImplementation((): any => {
+      return Promise.resolve(false)
```

Comment:
This was one bug: this call wasn't mocked and didn't return a value.

However, due to another bug, this was never called.